src/input/Input.js

```diff
@@ -111,11 +113,16 @@ class Input extends React.Component {
           <InputComponent
             testID="RNE__Input__text-input"
             underlineColorAndroid="transparent"
+            editable={!disabled}
             {...attributes}
             ref={ref => {
               this.input = ref;
             }}
-            style={StyleSheet.flatten([styles.input, inputStyle])}
+            style={StyleSheet.flatten([
+              styles.input,
+              inputStyle,
+              disabled && (disabledInputStyle || styles.disabledInput),
```

Comment:
Instead of choosing either or, we should merge.

See button for example https://github.com/react-native-training/react-native-elements/blob/master/src/buttons/Button.js#L103

Reply:
Makes sense. Thanks for the heads-up.